BLSQ/iaso
# IA-3312: User Right - "User management - Geo limited" I can not create user within the accessible part of the pyramide

Explain what problem this PR is resolving
- User Right - "User management - Geo limited" I can not create user within the accessible part of the pyramide
Related JIRA tickets : [IA-3312](https://bluesquare.atlassian.net/browse/IA-3312)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Let the user with User management - Geo limited to create, delete and bulk-create users

## How to test
- Create or Edit a user and add the **User management - Geo limited permission** without **User management - Admin**
- Check if the user can create, delete and bulk-create users

## Print screen / video
![Screenshot from 2024-08-26 19-40-06](https://github.com/user-attachments/assets/74f2b074-5830-4468-8516-1b276c1432f1)
![Screenshot-from-2024-08-26-19-41-29](https://github.com/user-attachments/assets/fe849f16-f0f6-402d-8eb1-465c5b31886a)


## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3312]: https://bluesquare.atlassian.net/browse/IA-3312?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
